job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/wrapper/WrapperContext.groovy

```diff
@@ -661,4 +661,24 @@ class WrapperContext extends AbstractExtensibleContext {
             jiraFilter(context.filter ?: '')
         }
     }
+
+    @RequiresPlugin(id = 'versionnumber')
+    void versionNumber(@DslContext(VersionNumberContext) Closure closure = null) {
+        VersionNumberContext context = new VersionNumberContext()
+        ContextHelper.executeInContext(closure, context)
+
+        wrapperNodes << new NodeBuilder().'org.jvnet.hudson.tools.versionnumber.VersionNumberBuilder' {
+            versionNumberString(context.versionNumberString ?: '')
+            projectStartDate(context.projectStartDate ?: new Date())
+            environmentVariableName(context.environmentVariableName ?: 'VERSION')
+            environmentPrefixVariable(context.environmentPrefixVariable ?: '')
+            oBuildsToday(context.oBuildsToday ?: '')
+            oBuildsThisWeek(context.oBuildsThisWeek ?: '')
+            oBuildsThisMonth(context.oBuildsThisMonth ?: '')
+            oBuildsThisYear(context.oBuildsThisYear ?: '')
+            oBuildsAllTime(context.oBuildsAllTime ?: '')
+            skipFailedBuilds(context.skipFailedBuilds ?: true)
+            useAsBuildDisplayName(context.useAsBuildDisplayName ?: true)
```

Comment:
You don't need to set any default values as that particular variable is already initialised